data/gala.gschema.xml

```diff
@@ -69,6 +70,11 @@
         followed by a ':' and then the command. ';;' is used as delimiter between command definitions.
       </description>
     </key>
+    <key type="b" name="enable-hotcorners-in-fullscreen">
```

Comment:
Ah sorry one more thing, can we add this to the `io.elementary.desktop.wm` namespace instead of to `org.pantheon.desktop.gala`? At some point I think we'll want to break this so that we're namespacing all the settings correctly, but at least a new setting it would be nice to have in the new namespace so we don't have to break it later too

Reply:
Done